feat(content): Korath Exiles Expansion, Revised - Stage I (#6891)
endless-sky/endless-sky: data/korath/korath outfits.txt

	"cooling" 54
	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."

outfit "Liquid Sodium Cooler"
	category "Systems"
	cost 1660000
	thumbnail "outfit/liquid sodium"
	"mass" 37
	"outfit space" -37

> 	"energy consumption" 1.5

Reviewer: ```suggestion
	"energy consumption" 3
```

> 	"cooling energy" 6

Reviewer: ```suggestion
	"cooling energy" 7.1
```

	"cooling" 54
	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."

outfit "Liquid Sodium Cooler"
	category "Systems"
	cost 1660000
	thumbnail "outfit/liquid sodium"
	"mass" 37

> 	"outfit space" -34

Reviewer 2: I don't really mind as long as there is some reason for it.

> 	"outfit space" -34

Author: Well, the reason I like to do these things is because, when left unexplained, it makes the player go "Oh, whoa, these high-tech outfits are weird," creating a sense of wonder in a very minor way.

	"cooling" 54
	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."

> outfit "Liquid Sodium Cooler"

Author: Yeah, I really think it is.  Especially when you're running more than just a TPC, with the new heat-based turret weapons, on Korath hulls with their heat dissipation in the .4-.5 range, and your primary military opponent operates in the Ember Wastes.  Or for the upcoming city-ships which are trying to cram as many souls as possible on board while still being able to escape from danger.

A lot of work from a lot of people went in to the balancing of these stats.  We've been very careful not to step on the Coalition's toes; this doesn't outright outclass a cooling module, but offers a different balance point.  That, and at 34 tonnes, over 9000 cooling, and its constant demand, it's too big and expensive to be worthwhile unless you *are* running a TPC.

If you're running LRMs, Blue Suns, or even a DPC, use Coal or Ka'het cooling.  If you're running anything other than a TPC, for that matter, use Coal or Ka'het cooling.  Even when you have a TPC, in an otherwise cool ship, use Coal or Ka'het cooling.  If you've got a TPC on top of a lot of other heat problems, or are rutting additional plasma cores on top of your TPC, that's when the Liquid Sodium Cooler becomes an *option*.  Not necessarily the best one for captains mixing outfits from different factions, but better that piling on umpteen heat shunts.

As for the sprite, I really like the new version, so I'm not sure what's bothering you about it.

	category "Systems"
	cost 1660000

> 	thumbnail "outfit/liquid sodium"

Author: Oh... okay, that's easy enough.  Though we did already start down a different direction

![lsc3](https://user-images.githubusercontent.com/31712538/180998028-37b816ec-36ce-4d59-ae13-428c73b9f378.png)

> 	thumbnail "outfit/liquid sodium"

Reviewer 3: Well now that one looks too dark to me. :)

> 	thumbnail "outfit/liquid sodium"

Author: Heh, we weren't sure whether it was the brightness, the yellowness, or the roundness, so I went after all three.

This is by no means a finished sprite, either. I can Blender 2.79 just fine, but the interface in newer versions drives me nuts, and this is a 2.8 blend, so this is just the result of me frustratedly banging my head against an interface I can't navigate in order to convey to our more expert modellers what ideas I've got so *they* can finish it.

	"mass" 34
	"outfit space" -34
	"energy consumption" 2

> 	"cooling energy" 6

Author: That earlier value you're referencing, 7.1 is from when it was also even bigger and did even more cooling... and that was too much consumption.  I'd rather redo this completely, maybe have it do `heat capacity` or some combination thereof, than push the energy figures up.

Also I don't think it makes sense to try and ensure a 5 tonne T1.5 outfit is more efficient than a 34 tonne T2 outfit.  Maybe the Heliarchs ought to have their own large cooling outfit that outshines this?

	"active cooling" 151

> 	description "This active cooling system pumps liquid sodium through the hottest parts of the ship. A secondary temperature control system draws a moderate amount of power in order to keep the sodium from freezing when the system is inactive."

Reviewer 4: The awkward thing about this is that the ship itself isn't really all that hot - or the people would all be dead. It's the critical systems that are hot, and "overheating' a ship conceptually just means warming up its exterior hull enough that the heat-dumping into space is no longer efficient enough to maintain cooling to critical systems.

Also, since lithium doesn't melt until ~180 Celsius unless kept under pressure, and room temperature melting of Lithium requires pressurising to between 40 and 60 GPa (which is a lot for any significant volume of anything - like murder everything in the room and probably puncture the hull with flash-frozen shards if it ruptures) it might be good to call out something involved in that.

I would suggest adding base-heat generation to it equal to what would consume 2 points of cooling energy.
Now you could go and change it so that instead of using energy consumption, you add that energy to the cooling energy which would feel intuitive if heat made sense - but this would mean it costs no power when running below 0 heat. 
Though you could also split it, and have a mix of consumption + heat with suitable added cooling energy if you wanted to make a nod to that.
Either way add the same amount of heat generation to the active cooling amount, and then this is a net 0 impact on performance when you're above 0 heat, but if you're using this to augment passive cooling it then uses up some of the passive cooling buffer. 

Accordingly I would suggest maybe something more like this: 
```suggestion
	description "This active cooling system uses a liquid sodium cooling loop to extract heat from the hottest ship systems. A secondary system draws a moderate amount of power to continuously run the pump and maintain a minimum temperature in order to keep the sodium from freezing when the system is inactive."
	description "	As a side effect of this, the system actually produces heat when idling. This raises the base operating heat of the ship, but the efficiency benefits at high heat levels are well-realized."`
```

Author: It's sodium, not lithium, which looks like it has a melting point of 98 C.  Not that that makes a difference.

I'm concerned that, after all the many rounds of carefully balancing this outfit with the input of so many people, that adding more functionality for simulation reasons at this point would hazard upsetting that carefully-obtained disposition and result in further arduous rounds of balancing and debate.

Your lore suggestion is spot-on, though.

Reviewer 4: Oh wow my brain really saw _**li**_ -quid sod- _**ium**_ and inserted lithium there... let me go clean my glasses.

Fair enough on the balance point - I'm like 98% sure the net impact could be made to be 0, but I fully respect that if the current balance works it may not be worth messing with just for lore fun, especially since _sodium's_ melting point is significantly lower.

If at some point in the future we end up with a suite of liquid-metal cooled active coolers across a few factions though, I do think it'd be a worthy exercise in that hypothetical future.

Happy the rest of it suited you. :)

> 	description "This active cooling system uses a liquid sodium cooling loop to extract heat from the hottest ship systems. A secondary system draws a moderate amount of power to continuously run the pump and maintain a minimum temperature in order to keep the sodium from freezing when the system is inactive."

Reviewer 5: ```suggestion
	description "This active cooling system uses a liquid sodium cooling loop to extract heat from the ship's hottest systems. A secondary system draws a moderate amount of power to continuously run the pump and maintain a minimum temperature in order to keep the sodium from freezing when the system is inactive."
```

outfit "Systems Core (Tiny)"
	plural "Systems Cores (Tiny)"
	category "Systems"
	cost 621000
	thumbnail "outfit/tiny systems core"
	"mass" 17
	"outfit space" -17
	"energy capacity" 1300
	"shield generation" 0.8
	"shield energy" 1.2
	"hull repair rate" 0.1
	"hull energy" 0.1
	"hull heat" 0.15

> 	description `Most Korath pilots dispense with Systems Cores, preferring to run larger than necessary generators and return to their mothership if they take damage. Few are willing to sacrifice enough firepower to mount this shield generator, capacitor, swarm of repair microbots, and computer to control them.`

Author: I suppose I could clarify that *military* or *attack* pilots don't use this, but *civilian* or *mining* pilots do.

	"cooling" 54
	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."

outfit "Liquid Sodium Cooler"
	category "Systems"
	cost 1660000
	thumbnail "outfit/liquid sodium"
	"mass" 34
	"outfit space" -34

> 	"heat capacity" 680

Reviewer 3: I do like the idea of the Korath using heat capacity. It's as if they've given up on getting rid of the heat entirely and instead are just looking for more places to put it. Admittedly given that this is such a new attribute though, I'm not sure on how effective it'll be; these values could potentially be too high, but I'll let them go for now and see how things play out.

Reviewer 6: I agree, it adds some nice differentiation from the usual "just try to get cold" all the time.

Reviewer: >I'm not sure on how effective it'll be; these values could potentially be too high, but I'll let them go for now and see how things play out.

I played around with it for a good while and it indeed is a bit weird compared to standard approaches to cooling, but in general, even with switching up it's weighting it's generally a trade-off of much higher base running heat % but eliminating the risk of overheating on your own, at the cost of a decent chunk of even Korath energy generation.

It ends up being more a supplement than a larger replacement cooling. Running one instead of an equal weight in heat shunts is generally a win, as long as you're still running at least a few heat shunts alongside it, and I liked how that felt. :)

Author: From what I gathered in the Discord discussion last night that rather went over my head, it sounds like this is also offering slightly better protection against heat attacks than active cooling, at the expense of constant energy use rather than scaling

Reviewer 2: @Amazinite in terms of effectiveness, `6 * "heat capacity" * "heat dissipation"` gives the amount by which it will increase 'max heat' (as displayed in game), so it would be equivalent to that much cooling (except it doesn't actually reduce the heat in the ship so it is no less vulnerable to infrared tracking).
On a Raider, 680 heat capacity is equivalent to 2040 cooling (heat dis = 0.5). On something like a Mereti ship, or a very small ship, it could be over 4000.

Using 0.5 heat dissipation:

name | creator | size | cooling | active cooling | cooling energy | energy consumption | heat cap | cooling/ton | active/ton | total/ton | active/energy
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Small Heat Shunt | Korath | 7 | 13.5 | 0 | 0 | 0 | 0 | 1.92857142857143 | 0 | 1.92857142857143 | 0
Large Heat Shunt | Korath | 24 | 54 | 0 | 0 | 0 | 0 | 2.25 | 0 | 2.25 | 0
Thermoelectric Cooler | Remnant | 4 | 0 | 8 | 0.4 | 0 | 0 | 0 | 2 | 2 | 20
Cooling Module | Coalition | 5 | 0 | 21 | 1.4 | 0 | 0 | 0 | 4.2 | 4.2 | 15
Large Sodium Cooler | Korath Exiles | 34 | 0 | 116 | 5.2 | 2.4 | 680 | 0 | 3.41176470588235 | 3.41176470588235 | 15.2631578947368
Large Sodium Cooler effective (0.5 heat dissipation) | Korath Exiles | 34 | 0 | 150 | 7.6 | 0 | 0 | 0 | 4.41176470588235 | 4.41176470588235 | 19.7368421052632

Considering the active cooling component alone (116 for 5.2 energy), we get an efficiency of just over 22 cooling/energy.
We can regard the heat capacity and energy consumption as always on active cooling. 680 heat capacity is equivalent to 34 cooling at 0.5 heat dissipation. 34/2.4 = ~14.2.

> 	"heat capacity" 1600

Reviewer: ```suggestion
	"heat capacity" 680
```

> 	"energy consumption" 4

Reviewer: ```suggestion
	"energy consumption" 2.4
```

> 	"cooling energy" 4

Reviewer: ```suggestion
	"cooling energy" 5.2
```

> 	"active cooling" 75

Reviewer: ```suggestion
	"active cooling" 116
```

	"hull repair rate" 0.2
	"hull energy" 0.2
	"hull heat" 0.3

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`

Author: Technically, yes, but practically, no.  Usually when we say "small number" we mean something something easily counted by a human, these are thousands of sand-grain sized robots that would be more convenient to measure by volume than absolute quantity, and "small amount" implies this better.

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`

Author: (also if anyone is wondering and not privy to the discussion on Discord, there were so many different descriptions and even names for the bots the Korath use, that I decided to standardise them on how the Remnant describe them, as microbots, because A. that's the only mission content where they are relevant, and B. to distinguish them from the *nano*bots of Sayiban infamy.)

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`

Reviewer 2: Then, perhaps it should be `large number`, or specifically say `small volume`.

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`

Author: I really think that's too specific and doesn't sound right.

If someone gave you a jar of sand, would you say they gave you a large number or small volume of sand grains?  Only if you were being silly.  Rather, you would say you received a small amount of sand.

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`

Author: I'm not sure where you get that idea at all.  If I give you a jar of sand, and a jar of sand-sized microbots, what makes the former uncountable and the latter countable?

Either way it's a moot point.  Countable or not, `amount` is an appropriate descriptor.  Say I gave you a small pile of cash, you could just as well say I gave you a small amount of cash as you could count it and say I gave you $50.12.  Except cash is very easy to count.

Ultimately, this is a stylistic choice, not a grammar issue, and I find amount to be more aesthetically pleasing than number.

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`

Reviewer 6: My two cents here is that it depends on the speaker. If I had a generic crew member talking about them in a high-level general sense, I'd probably have them say. "Cores usually have a small number of microbots." If the same generic crew was talking about a specific core, they'd probably be more likely to say. "This particular core has a small amount of microbots available to repair the ship." If I had Taely or a science officer talking about them, they'd probably opt for the more scientific "This small system core contains a small volume of microbots."

In other words, I find "number" and "amount" and "volume" to be basically synonymous, and differing mainly in how I want to portray the speaker. 

**tldr:** I don't think number vs amount vs volume has sufficient differentiation to be worth protracted discussion on. It's good as is, so I would suggest just leaving it as is.

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`

Author: > **tldr:** I don't think number vs amount vs volume has sufficient differentiation to be worth protracted discussion on. It's good as is, so I would suggest just leaving it as is.

Me neither, but I think there is a key component missing to this that gives me a chance to spin lore at y'all -- and that is that there is not a fixed number of microbots in a systems core.  A systems core, as we see with the Shunt-Strike, both controls and *produces* microbots.  Inevitably these little guys are going to get blasted off the hull while they're trying to repair it under fire, or get flung off during high-G manoeuvres, so it has to keep producing them -- and, there lies another key difference between the microbots and nanobots, nanobots self-replicate whereas microbots are produced by a larger device.

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`

Author: As someone who dabbles in metalwork I get that welding sounds too specific :-)

Walking sounds clumsy to me though.  Swarming, as you put it, is evocative, visually stimulating, and interesting, walking sounds like a clumsy thing.  Maybe the Coal repair bots walk, because we can actually see they're quite big fellows, but walking doesn't sound like the right verb to me for microbots.

Not opposed to removing the stipulation about "minor repairs to the hull," though if you want to be honest, that's all that ever gets damaged on your ship in ES.

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`

Reviewer 6: Sure, swapping out swarming for walking sounds good.

	"hull repair rate" 0.2
	"hull energy" 0.2
	"hull heat" 0.3

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small microbot swarm that can make minor repairs to the ship while in flight.`

Reviewer 4: Adding formally to this review: Since there's no indication that the swarm can expire or wear out I would suggest that it actually produces the swarm through some sort of 3D printing technology (something probably similar to stereolithography or DMLS):
```suggestion
	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, reserve power, and a small reservoir which stores, produces and recycles a small microbot swarmin that can make minor repairs while in flight.`
```

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-swarming microbots that can make minor repairs while in flight.`

Reviewer 4: Since there's no indication that the swarm can expire or wear out I would suggest that it actually *produces* the swarm through some sort of 3D printing technology (something probably similar to stereolithography or DMLS):
```suggestion
	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, reserve power, and a small reservoir which stores, produces and recycles hull-swarming microbots that can make minor repairs while in flight.`
```

> 	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-swarming microbots that can make minor repairs while in flight.`

Author: That's a great suggestion, but I think I'll add it to the more commonly-encountered Medium variant.

	"hull repair rate" 0.4
	"hull energy" 0.4
	"hull heat" 0.6

> 	description `Systems cores house a powerful computer, shield generators, and a control center for repair microbots that can move across a ship's exterior and slowly repair damage to the hull.`

Reviewer 6: Hmmmm.... That's weird, because that *really* doesn't sound correct to me.

> 	description `Systems cores house a powerful computer, shield generators, and a control center for repair microbots that can move across a ship's exterior and slowly repair damage to the hull.`

Reviewer 2: I had assumed it referred to it being a single 'core' containing multiple different systems.

> 	description `This "systems core" houses a powerful computer, shield generators, and a control center for repair microbots that can move across a ship's exterior and slowly repair damage to the hull.`

Author: Medium sized seems a bit redundant.  I could see dropping the quotes and replacing This with The, though.

	"hull repair rate" 0.4
	"hull energy" 0.4
	"hull heat" 0.6

> 	description `The Systems core houses a powerful computer, shield generators, and a control center for repair microbots that can move across a ship's exterior and slowly repair damage to the hull.`

Reviewer 4: ```suggestion
	description `The Systems Core houses a powerful computer, shield generators, reserve power, and a control center for repair microbots that can move across a ship's exterior and slowly repair damage to the hull. The reservoir it has for storing, producing and recycling microbots is capable of supporting a significant swarm.`
```

	"hull repair rate" 0.8
	"hull energy" 0.8
	"hull heat" 1.2

> 	description `This massive bank of computers, shield generators, and repair microbot stations holds everything needed to keep the very largest of Korath ships in working condition.`

Reviewer 4: ```suggestion
	description `This massive bank of computers, shield generators, capacitors and repair microbot reservoirs holds everything needed to keep the very largest of Korath ships in working condition.`
```

	description "This is a miniaturized communication device, which can be connected to all the major systems of a Kor Sestor automaton, apparently as a way of controlling them remotely. The communications are encrypted and authenticated in a way that would make it nearly impossible to spoof the control signals."

> outfit "Afterburner (Asteroid Class)"

Reviewer 7: I just have a problem balance wise, this is way too weak in my opinion.
Okay this may provide ramscoop, and be efficient but still just compare these stats to the 10 sized human afterburner, which thrusts 5 times as much and is only 2.5 times less efficient...
I'm not even speaking about the remnant ones for which the smallest size 7 is better than the size 55 version of this thing (it uses 15 times as much fuel though but still, size 7); I know its the thing of remnants to have strong afterburners, but still I feel like this one needs to be buffed.
Either we make it even more efficient and really be an afterburner that is close to an actual engine in thrust, or we buff its thrust.
But when it is said this nearly explodes my engines I expect this thing to be at least 2x as strong than what it is currently.

	plural "Afterburners (Asteroid Class)"
	category "Engines"
	"cost" 136000
	thumbnail "outfit/tiny korath afterburner"
	"mass" 10
	"outfit space" -10
	"engine capacity" -10
	"fuel capacity" 100
	"ramscoop" 0.5
	"afterburner thrust" 8.175
	"afterburner energy" 0.77
	"afterburner heat" 6.26

> 	"afterburner fuel" 0.0408

Reviewer 7: I just have a problem balance wise, this is way too weak in my opinion.
Okay this may provide ramscoop, and be efficient but still just compare these stats to the 10 sized human afterburner, which thrusts 5 times as much and is only 2.5 times less efficient...
I'm not even speaking about the remnant ones for which the smallest size 7 is better than the size 55 version of this thing (it uses 15 times as much fuel though but still, size 7); I know its the thing of remnants to have strong afterburners, but still I feel like this one needs to be buffed.
Either we make it even more efficient and really be an afterburner that is close to an actual engine in thrust, or we buff its thrust.
But when it is said this nearly explodes my engines I expect this thing to be at least 2x as strong than what it is currently.

Reviewer 2: I think doubling the thrust, energy, and heat would be a good start.

Author: Actually, just doubling the thrust makes them competitive with Human afterburners.  Stellar-class is now better than Caldera, while Comet and Asteroid are worse than Volcano, on a ton-for-ton basis, all while Korath burners are terrible on heat.

	"mass" 94
	"outfit space" -94
	"engine capacity" -94
	"afterburner thrust" 460
	"afterburner energy" 15.8
	"afterburner heat" 149.1
	"afterburner fuel" 0.863
	"afterburner effect" "korath afterburner"
	"flare sound" "plasma medium"
	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenominal explosion that nearly melts the thruster."
	description "The very largest of Korath afterburners have integrated active cooling systems, powered by the fuel-plasma explosion, to keep the reaction chambers from melting."

effect "korath afterburner"
	sprite "effect/korath afterburner"
		"no repeat"
		"frame rate" 55
	sound "korath afterburner"
	"lifetime" 10
	"random angle" 15
	"random spin" 360
	"random frame rate" 5
	"random velocity" 1.
	"velocity scale" 1

outfit "Engine (Meteor Class)"
	plural "Engine (Meteor Class)"
	category "Engines"
	"cost" 187000
	thumbnail "outfit/tiny korath engine"
	"mass" 18
	"outfit space" -18
	"engine capacity" -18
	"thrust" 7.5
	"turn" 220
	"thrusting energy" 1.3
	"thrusting heat" 2.9
	"turning energy" 0.6
	"turning heat" 1.8
	"flare sprite" "effect/korath flare/tiny"
		"frame rate" 8

> 	"flare sound" "plasma tiny"

Reviewer: ```suggestion
	"flare sound" "plasma tiny"
	"steering flare sprite" "effect/korath flare/tiny"
		"frame rate" 8
	"steering flare sound" "plasma tiny"
```

	"reverse thrusting heat" 3.4
	"reverse flare sprite" "effect/korath flare/tiny"
		"frame rate" 8
	"reverse flare sound" "plasma tiny"
	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."

outfit "Reverser (Comet Class)"
	plural "Reversers (Comet Class)"
	category "Engines"
	"cost" 249000
	thumbnail "outfit/small korath reverser"
	"mass" 24
	"outfit space" -24
	"weapon capacity" -24
	"reverse thrust" 21.8
	"reverse thrusting energy" 2.8
	"reverse thrusting heat" 6.5
	"reverse flare sprite" "effect/korath flare/small"
		"frame rate" 7.5
	"reverse flare sound" "plasma small"
	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."

outfit "Reverser (Lunar Class)"
	plural "Reversers (Lunar Class)"
	category "Engines"
	"cost" 539000
	thumbnail "outfit/medium korath reverser"
	"mass" 40
	"outfit space" -40
	"weapon capacity" -40
	"reverse thrust" 41.2
	"reverse thrusting energy" 4.9
	"reverse thrusting heat" 11.9
	"reverse flare sprite" "effect/korath flare/medium"
		"frame rate" 7
	"reverse flare sound" "plasma medium"

> 	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."

Author: Of course I agree, and would welcome anyone who'd like to provide such.

		"frame rate" 8
	"reverse flare sound" "plasma tiny"
	"steering flare sound" "plasma tiny"
	description "This modification of the Meteor Class engines acts as a reverser, allowing a full range of mobility for small craft when paired with more powerful base thrusters."

outfit "Reverser (Asteroid Class)"
	plural "Reverser (Asteroid Class)"
	category "Engines"
	"cost" 113000
	thumbnail "outfit/tiny korath reverser"
	"mass" 14
	"outfit space" -14
	"weapon capacity" -14
	"reverse thrust" 11.2
	"reverse thrusting energy" 1.5
	"reverse thrusting heat" 3.4
	"reverse flare sprite" "effect/korath flare/tiny"
		"frame rate" 8
	"reverse flare sound" "plasma tiny"
	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."

outfit "Reverser (Comet Class)"
	plural "Reversers (Comet Class)"
	category "Engines"
	"cost" 249000
	thumbnail "outfit/small korath reverser"
	"mass" 24
	"outfit space" -24
	"weapon capacity" -24
	"reverse thrust" 21.8
	"reverse thrusting energy" 2.8
	"reverse thrusting heat" 6.5
	"reverse flare sprite" "effect/korath flare/small"
		"frame rate" 7.5
	"reverse flare sound" "plasma small"

> 	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."

Reviewer 8: Is the updated suggestion better?

outfit "Reverser (Lunar Class)"
	plural "Reversers (Lunar Class)"
	category "Engines"
	"cost" 539000
	thumbnail "outfit/medium korath reverser"
	"mass" 40
	"outfit space" -40
	"weapon capacity" -40
	"reverse thrust" 41.2
	"reverse thrusting energy" 4.9
	"reverse thrusting heat" 11.9
	"reverse flare sprite" "effect/korath flare/medium"
		"frame rate" 7
	"reverse flare sound" "plasma medium"

> 	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."

Reviewer 8: ```suggestion
	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."
	description "Dwarfing any reversers in human space, this engine provides a considerable amount of reverse thrust for any captain willing to forgo the weapons space it takes up."
```

Author: Colourful, though, after the comma, I might instead drive the point home by mentioning that it's commonly used on medium-sized Korath freighters

Reviewer 8: I changed my suggestion, is it better?

outfit "Reverser (Planetary Class)"
	plural "Reversers (Planetary Class)"
	category "Engines"
	"cost" 1216000
	thumbnail "outfit/large korath reverser"
	"mass" 69
	"outfit space" -69
	"weapon capacity" -69
	"reverse thrust" 80.0
	"reverse thrusting energy" 8.9
	"reverse thrusting heat" 22.7
	"reverse flare sprite" "effect/korath flare/large"
		"frame rate" 6.5
	"reverse flare sound" "plasma large"

> 	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."

Reviewer 8: ```suggestion
	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."
	description "Being essentially a slightly altered Planetary Class engine, this reverser provides an enormous amount of reverse thrust for any ship capable of installing it."
```

	description "This is a miniaturized communication device, which can be connected to all the major systems of a Kor Sestor automaton, apparently as a way of controlling them remotely. The communications are encrypted and authenticated in a way that would make it nearly impossible to spoof the control signals."

outfit "Afterburner (Asteroid Class)"
	plural "Afterburners (Asteroid Class)"
	category "Engines"
	"cost" 136000
	thumbnail "outfit/tiny korath afterburner"
	"mass" 10
	"outfit space" -10
	"engine capacity" -10
	"fuel capacity" 100

> 	"ramscoop" 0.5

Reviewer: This makes it a better fuel carrier than the independent fuel pod, with the added benefit of some afterburner in exchange for solely engine space. With the ramscoop included it's a not much much smaller but much less efficient fuel processor with the afterburner benefits.

Removing ramscoop would also suggest a remodel, since that includes parts directly appealing to the other Exile outfits with that tech.

A volcano is stronger afterburner, yes, but it would eat up (at least) 8 more of a fighter's outfit space.

> 	"ramscoop" 0.5

Author: RIP the World-Ship Chaser if you remove the ramscoop, which is the only "fighter" that's actually been using this so far.  In some variants, I've been throwing this on specifically for the ramscoop rather than the thrust, when there's not enough space for a Fuel Processor.

How about keeping the ramscoop, but reducing the thrust, or possibly, carefully, increasing the energy, heat, or fuel consumption?

> 	"ramscoop" 0.5

Reviewer 6: Alright,  I see the changes you made, and I think it looks good. I'd still like to suggest dropping the space and mass, though. Even if it is just down to 9, I'd like to see just a little more density than it currently does. And have it be a *differently* shaped piece in the outfitting game, rather than be identically sized to the volcano.

> 	"ramscoop" 0.5

Author: Nine is fine.  I doubt the similarity was intentional; at the time of its conception, there was no Volcano afterburner, just the plain Afterburner which was IIRC 15 tonnes.

	category "Engines"
	"cost" 647000
	thumbnail "outfit/medium korath afterburner"
	"mass" 32
	"outfit space" -32
	"engine capacity" -32
	"afterburner thrust" 124
	"afterburner energy" 4.9
	"afterburner heat" 41.65
	"afterburner fuel" 0.18
	"afterburner effect" "korath afterburner"
	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
	description "Although these afterburners produce an incredible amount of thrust for their fuel usage, you can't help but feel concerned about the safety of installing one."

outfit "Afterburner (Planetary Class)"
	plural "Afterburners (Planetary Class)"
	category "Engines"
	"cost" 1459000
	thumbnail "outfit/large korath afterburner"
	"mass" 55
	"outfit space" -55
	"engine capacity" -55
	"afterburner thrust" 240
	"afterburner energy" 8.9
	"afterburner heat" 79.45
	"afterburner fuel" 0.32
	"afterburner effect" "korath afterburner"
	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."

> 	description "These enormous afterburners have dedicated reaction chambers with cooling vents to keep the fuel-plasma explosion from destroying the thruster."

Author: note to self: merge in batch when the files changed page finally loads

outfit "Afterburner (Stellar Class)"
	plural "Afterburners (Stellar Class)"
	category "Engines"
	"cost" 3280000
	thumbnail "outfit/huge korath afterburner"
	"mass" 94
	"outfit space" -94
	"engine capacity" -94
	"afterburner thrust" 460
	"afterburner energy" 15.8
	"afterburner heat" 149.1
	"afterburner fuel" 0.58
	"afterburner effect" "korath afterburner"
	"flare sound" "plasma medium"
	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."

> 	description "The very largest of Korath afterburners have integrated active cooling systems, powered by the fuel-plasma explosion, in order to keep the reaction chambers from melting."

Author: Worth taking a look at though!  Hopefully someone more stat wise can come up with something, or I'll get to it later

> 	description "The very largest of Korath afterburners have integrated active cooling systems, powered by the fuel-plasma explosion, in order to keep the reaction chambers from melting."

Reviewer: I had always figured it as already included in the energy and heat costs - the one would be higher and the other lower if not cooling wasn't activated alongside afterburner.

Maybe they can be adjusted off the curves for the others to highlight that more though?

	description "This is a miniaturized communication device, which can be connected to all the major systems of a Kor Sestor automaton, apparently as a way of controlling them remotely. The communications are encrypted and authenticated in a way that would make it nearly impossible to spoof the control signals."

outfit "Afterburner (Asteroid Class)"
	plural "Afterburners (Asteroid Class)"
	category "Engines"
	"cost" 136000
	thumbnail "outfit/tiny korath afterburner"

> 	"mass" 10
> 	"outfit space" -10

Reviewer 6: ```suggestion
	"mass" 9
	"outfit space" -9
```

The other suggestion for changes here became outdated, so creating a new suggestion for it. This merely reduces the mass and outfit space by 1, which improves the Korath utility-density slightly, and gives it a different use profile from the human afterburner at size 10. As in there may be some ships that can't fit that afterburner, but could fit this one. And having such diversity improves the game.

	description "These enormous afterburners have dedicated reaction chambers with cooling vents to keep the fuel-plasma explosion from destroying the thruster."

outfit "Afterburner (Stellar Class)"
	plural "Afterburners (Stellar Class)"
	category "Engines"
	"cost" 3280000
	thumbnail "outfit/huge korath afterburner"
	"mass" 94
	"outfit space" -94
	"engine capacity" -94
	"afterburner thrust" 552
	"afterburner energy" 12.64
	"afterburner heat" 148
	"afterburner fuel" 0.58
	"afterburner effect" "korath afterburner"
	"heat capacity" 900
	"cooling energy" 0.4
	"active cooling" 8
	"flare sound" "plasma medium"
	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
	description "The very largest of Korath afterburners have integrated active cooling systems, powered by the fuel-plasma explosion, in order to keep the reaction chambers from melting."

effect "korath afterburner"
	sprite "effect/korath afterburner"
		"no repeat"
		"frame rate" 55
	sound "korath afterburner"
	"lifetime" 10
	"random angle" 15
	"random spin" 360
	"random frame rate" 5
	"random velocity" 1.

> 	"velocity scale" 1

Reviewer 9: Same with this as the Heaver thing. It's a bit out of scope for the current PR as it's content-frozen for additions like this and review ready.

However, if Raven wants to do this, we don't need any new code or anything, just different afterburner effects using a different ```scale```.

> 	"velocity scale" 1

Author: No, that's a great suggestion.  If `scale` will do the trick I'll certainly do that rather than make more sprites though, as it's less of a change for this late stage.

> 	"velocity scale" 1

Reviewer 4: Scale should do it, I would recommend not scaling up too much, but scaling down should be nice and safe.

	description "This is a miniaturized communication device, which can be connected to all the major systems of a Kor Sestor automaton, apparently as a way of controlling them remotely. The communications are encrypted and authenticated in a way that would make it nearly impossible to spoof the control signals."

outfit "Afterburner (Asteroid Class)"
	plural "Afterburners (Asteroid Class)"
	category "Engines"
	"cost" 136000
	thumbnail "outfit/tiny korath afterburner"
	"mass" 9
	"outfit space" -9
	"engine capacity" -5
	"fuel capacity" 100
	"ramscoop" 0.25
	"afterburner thrust" 19.2
	"afterburner energy" 0.616
	"afterburner heat" 6.26
	"afterburner fuel" 0.03
	"afterburner effect" "korath afterburner tiny"
	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that may nearly melt the thruster."

> 	description "The smallest of afterburners is as much fuel processor as it is afterburner. It allows Korath small craft operators to squeeze a hyperdrive into otherwise intrastellar ships and dispense with conventional thrusters entirely."

Author: Hm, it seems you are correct, intrastellar refers to within a star.
How about
```suggestion
	description "The smallest of afterburners is as much fuel processor as it is afterburner. It allows Korath small craft operators to squeeze a hyperdrive into otherwise intrasystem ships and dispense with conventional thrusters entirely."
```

	thumbnail "outfit/small korath afterburner"
	"mass" 19
	"outfit space" -19
	"engine capacity" -19
	"afterburner thrust" 78
	"afterburner energy" 2.24
	"afterburner heat" 22.75
	"afterburner fuel" 0.1
	"afterburner effect" "korath afterburner small"
	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
	description "Adding these afterburners to an interceptor allows it to reach absurd speeds; however, the heat produced ensures that running them for any substantial period of time will require some serious cooling."

outfit "Afterburner (Lunar Class)"
	plural "Afterburners (Lunar Class)"
	category "Engines"
	"cost" 647000
	thumbnail "outfit/medium korath afterburner"
	"mass" 32
	"outfit space" -32
	"engine capacity" -32
	"afterburner thrust" 148.8
	"afterburner energy" 3.92
	"afterburner heat" 41.65
	"afterburner fuel" 0.18
	"afterburner effect" "korath afterburner medium"
	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."

> 	description "Although these afterburners produce an incredible amount of thrust for their fuel usage, you can't help but feel concerned about the safety of installing one."

Reviewer 10: ```suggestion
	description "Although these afterburners produce an incredible amount of thrust for their fuel usage, the prospect of installing one brings to mind many safety concerns."
```
It's best to avoid pronouns in descriptions, I think. This feels a tiny bit better to me, at least.

outfit "Afterburner (Planetary Class)"
	plural "Afterburners (Planetary Class)"
	category "Engines"
	"cost" 1459000
	thumbnail "outfit/large korath afterburner"
	"mass" 55
	"outfit space" -55
	"engine capacity" -55
	"afterburner thrust" 288
	"afterburner energy" 7.12
	"afterburner heat" 79.45
	"afterburner fuel" 0.32
	"afterburner effect" "korath afterburner large"
	"heat capacity" 500
	"heat dissipation" 0.02
	description "These enormous afterburners have dedicated reaction chambers with cooling vents to keep the fuel-plasma explosion from destroying the thruster."
	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."

outfit "Afterburner (Stellar Class)"
	plural "Afterburners (Stellar Class)"
	category "Engines"
	"cost" 3280000
	thumbnail "outfit/huge korath afterburner"
	"mass" 94
	"outfit space" -94
	"engine capacity" -94
	"afterburner thrust" 552
	"afterburner energy" 12.64
	"afterburner heat" 148
	"afterburner fuel" 0.58
	"afterburner effect" "korath afterburner huge"
	"heat capacity" 900
	"cooling energy" 0.4
	"active cooling" 8

> 	"flare sound" "plasma medium"

Author: To give it a little something more, since it's the largest one

> 	"flare sound" "plasma medium"

Reviewer 3: I'm pretty sure this sound will play whenever you're thrusting, not whenever you're using the afterburner, which seems odd to me that the afterburner would make noise while not in use.

> 	"flare sound" "plasma medium"

Author: ...Yep, after actually testing this, you're right, it doesn't play at the right time

> 	"flare sound" "plasma medium"

Reviewer: Is it just a matter of putting in the right attribute, or was this chosen hoping to work because there is no afterburner specific equivalent?

	"velocity scale" 1

effect "korath afterburner huge"
	sprite "effect/korath afterburner"
		scale 1.6
		"no repeat"
		"frame rate" 20
	sound "korath afterburner"
	"lifetime" 16
	"random angle" 15
	"random spin" 360
	"random frame rate" 5
	"random velocity" 1.
	"velocity scale" 1

outfit "Engine (Meteor Class)"
	plural "Engine (Meteor Class)"
	category "Engines"
	"cost" 187000
	thumbnail "outfit/tiny korath engine"
	"mass" 18
	"outfit space" -18
	"engine capacity" -18
	"thrust" 7.5
	"turn" 220
	"thrusting energy" 1.3
	"thrusting heat" 2.9
	"turning energy" 0.6
	"turning heat" 1.8
	"flare sprite" "effect/korath flare/tiny"
		"frame rate" 8
	"flare sound" "plasma tiny"
	"steering flare sprite" "effect/korath flare/tiny"
		"frame rate" 8
	"steering flare sound" "plasma tiny"
	description `Although the prevailing Korath attitude towards engine construction has always been with a focus on making things bigger, faster, and more powerful, the Exiles found it necessary to develop these engines to allow their fighters to carry larger weapons or more cargo, at the cost of some speed.`

outfit "Bow Drive (Meteor Class)"
	plural "Bow Drives (Meteor Class)"
	category "Engines"
	"cost" 187000

> 	thumbnail "outfit/korath bow drive"

Reviewer 10: The name "Bow Drive" is nice, but the image itself looks a bit silly. It's a big triangle of thrusters! Korath fidget spinner! I'd rather the steering parts of the Bow Drive be turned to face forward with the reversing part, as if you had taken any old thruster+steering outfit and spun it around 180 degrees.

	description "This Korath engine has been modified to be partially installed in a ship's weapon space, allowing it to push the ship in reverse."

outfit "Reverser (Lunar Class)"
	plural "Reversers (Lunar Class)"
	category "Engines"
	"cost" 539000
	thumbnail "outfit/medium korath reverser"
	"mass" 40
	"outfit space" -40
	"engine capacity" -20
	"weapon capacity" -20
	"reverse thrust" 41.2
	"reverse thrusting energy" 4.9
	"reverse thrusting heat" 11.9
	"reverse flare sprite" "effect/korath flare/medium"
		"frame rate" 7
	"reverse flare sound" "plasma medium"
	description "Dwarfing any reversers in human space, this engine provides a considerable amount of reverse thrust for medium sized Korath ships, where it is most commonly found."
	description "This Korath engine has been modified to be partially installed in a ship's weapon space, allowing it to push the ship in reverse."

outfit "Reverser (Planetary Class)"
	plural "Reversers (Planetary Class)"
	category "Engines"
	"cost" 1216000
	thumbnail "outfit/large korath reverser"
	"mass" 69
	"outfit space" -69
	"engine capacity" -34

> 	"weapon capacity" -34

Reviewer 6: Any particular reason for not wanting an odd number here?

> 	"weapon capacity" -34

Reviewer 3: For all other reversers, engine capacity + weapon capacity = outfit space. Here, engine capacity + weapon capacity = outfit space - 1.

> 	"weapon capacity" -34

Reviewer 6: I believe the theory is that a sliver of the outfit takes up space that doesn't forcibly have to be exterior-facing weapon or engine space. I don't think a 1 ton difference is particularly important either way.

> 	"weapon capacity" -34

Reviewer 9: I would very much suggest removing it from outfit space as adding any mass will break a lot of ships and variants (which we can still fix of course, it's just a ton more work).

> 	"weapon capacity" -34

Author: Although no variants would be harmed in the sacrifice of 1 tonne of weapon space on this outfit, I agree with Zitchas.  It seems cleaner to me have the proportion of engine to weapon match, and for the mass to match with the thruster, than to have the reverser be smaller or the engine and weapon space to mismatch.

> 	"weapon capacity" -34

Reviewer 3: I was thinking of having the reverser and thruster both receive the same change, not just only change the reverser to 68 tons.

	"reverse thrust" 80.0
	"reverse thrusting energy" 8.9
	"reverse thrusting heat" 22.7
	"reverse flare sprite" "effect/korath flare/large"
		"frame rate" 6.5
	"reverse flare sound" "plasma large"
	description "Being essentially a slightly altered Planetary Class engine, this reverser provides an enormous amount of reverse thrust for any ship capable of installing it."
	description "This Korath engine has been modified to be partially installed in a ship's weapon space, allowing it to push the ship in reverse."

outfit "Reverser (Stellar Class)"
	plural "Reversers (Stellar Class)"
	category "Engines"
	"cost" 2733000
	thumbnail "outfit/huge korath reverser"
	"mass" 118
	"outfit space" -118
	"engine capacity" -58

> 	"weapon capacity" -58

Author: Will have to check whether it screws up variants before I merge this

outfit "Afterburner (Lunar Class)"
	plural "Afterburners (Lunar Class)"
	category "Engines"
	"cost" 647000
	thumbnail "outfit/medium korath afterburner"
	"mass" 32
	"outfit space" -32
	"engine capacity" -32
	"afterburner thrust" 148.8
	"afterburner energy" 3.9
	"afterburner heat" 41.65
	"afterburner fuel" 0.18
	"afterburner effect" "korath afterburner medium"
	description "Although these afterburners produce an incredible amount of thrust for their fuel usage, the prospect of installing one brings to mind many safety concerns."
	description "	Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."

outfit "Afterburner (Planetary Class)"
	plural "Afterburners (Planetary Class)"
	category "Engines"
	"cost" 1459000
	thumbnail "outfit/large korath afterburner"
	"mass" 55
	"outfit space" -55
	"engine capacity" -55
	"afterburner thrust" 288
	"afterburner energy" 7.1
	"afterburner heat" 79.45
	"afterburner fuel" 0.32
	"afterburner effect" "korath afterburner large"

> 	"heat capacity" 500

Reviewer: ```suggestion
	"heat capacity" 30
```

	description "These enormous afterburners have dedicated reaction chambers with cooling vents to keep the fuel-plasma explosion from destroying the thruster."
	description "	Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."

outfit "Afterburner (Stellar Class)"
	plural "Afterburners (Stellar Class)"
	category "Engines"
	"cost" 3280000
	thumbnail "outfit/huge korath afterburner"
	"mass" 94
	"outfit space" -94
	"engine capacity" -94
	"afterburner thrust" 552
	"afterburner energy" 12.65
	"afterburner heat" 148
	"afterburner fuel" 0.58
	"afterburner effect" "korath afterburner huge"

> 	"heat capacity" 900

Reviewer: ```suggestion
	"heat capacity" 62
```

Reviewer 6: Have these new values been tested? They look too low to be meaningful.

Reviewer: These two I haven't extensively yet, they're just bumping it down to match the cooling in the same proportion as the LSC, so these specific numbers are essentially just my best starting placeholder for "this needs adjusted."

Because after playing around with the LSC adjustments to settle on those numbers, the original capacity values for these two are definitely -far, far- too high.

I'd myself also still like to see these both adjusted to have higher energy but lower heat usage than would be in line with the other Korath afterburners because of the powered cooling on them instead of them actually granting general extra cooling when used, and any decision on that will probably influence exactly how much heat capacity ends up being appropriate anyway.

Reviewer 2: I think this value is very low. It would effectively provide a ship with a heat dissipation value of 0.5 with 186 cooling. A Mereti ship might get 350-400 effective cooling from this. That's not really a lot.

100 would be 300 cooling at 0.5 hear dissipation

Maybe around 150 for this one.

	"capture defense" 2.4
	"unplunderable" 1
	description "This rifle is commonplace among the Korath, predating the creation of the Repeater by at least two centuries. As it uses superheated plasma instead of lasers, the rifle is slow to fire and can sometimes burn its user's hands if they aren't careful. It makes up for these setbacks by guaranteeing that those hit won't live long enough to fire back."

outfit "Nanobot Defense Station"
	category "Hand to Hand"
	cost 53000
	thumbnail "outfit/nanobot defense station"
	mass 2
	"outfit space" -3
	"capture defense" 45
	atrocity 1
	"energy consumption" 0.01
	"heat generation" 0.1
	unplunderable 1
	description `This insidious trap awaits anyone foolish enough to attempt to capture Korath World-Ships.  Intruders are trapped before this airlock, ahead of which a cloud of nanobots programmed to break down organic matter into nutrigel is released.  The resulting gel then drains into the vat below while the robots are recalled.`

> 	description `Many races find this means of defense abhorrent.  Citing the excruciating, confusing death experienced by its victims; the violation of taboos relating to cannibalism, and erroneous fears that the nanobots could somehow "escape," fines are levied against anyone found with these airlocks.`  

Reviewer 7: it says fines are levied against anyone with these but it is considered an atrocity so that's way worse than a simple fine

Author: Good catch.  It was a fine to start with before Darcy told me about the new atrocity behaviour.

	"capture defense" 2.4
	"unplunderable" 1
	description "This rifle is commonplace among the Korath, predating the creation of the Repeater by at least two centuries. As it uses superheated plasma instead of lasers, the rifle is slow to fire and can sometimes burn its user's hands if they aren't careful. It makes up for these setbacks by guaranteeing that those hit won't live long enough to fire back."

outfit "Nanobot Defense Station"
	category "Hand to Hand"
	cost 53000
	thumbnail "outfit/nanobot defense station"
	mass 2
	"outfit space" -3
	"capture defense" 45
	atrocity 1
	"energy consumption" 0.01
	"heat generation" 0.1
	unplunderable 1
	description `This insidious trap awaits anyone foolish enough to attempt to capture Korath World-Ships.  Intruders are trapped before this airlock, ahead of which a cloud of nanobots programmed to break down organic matter into nutrigel is released.  The resulting gel then drains into the vat below while the robots are recalled.`

> 	description `Many races find this means of defense abhorrent.  Citing the excruciating, confusing death experienced by its victims; the violation of taboos relating to cannibalism, and erroneous fears that the nanobots could somehow "escape," mere possession of these airlocks may carry the death penalty.`

Reviewer 8: ```suggestion
	description `Many races find this means of defense abhorrent. Citing the excruciating, confusing death experienced by its victims, the violation of taboos relating to cannibalism, and erroneous fears that the nanobots could somehow "escape," mere possession of these airlocks may carry the death penalty.`
```

	"capture defense" 2.4
	"unplunderable" 1
	description "This rifle is commonplace among the Korath, predating the creation of the Repeater by at least two centuries. As it uses superheated plasma instead of lasers, the rifle is slow to fire and can sometimes burn its user's hands if they aren't careful. It makes up for these setbacks by guaranteeing that those hit won't live long enough to fire back."

outfit "Microbot Defense Station"
	category "Hand to Hand"
	cost 53000
	thumbnail "outfit/microbot defense station"
	mass 2
	"outfit space" -3
	"capture defense" 45
	atrocity 1
	"energy consumption" 0.01
	"heat generation" 0.1
	unplunderable 1

> 	description `This insidious trap awaits anyone foolish enough to attempt to capture Korath World-Ships. Intruders are trapped before this airlock, ahead of which a cloud of microbots programmed to break down organic matter into nutrigel is released.  The resulting gel then drains into the vat below while the robots are recalled.`

Author: Yeah, I learned to type on a typewriter.... and in most contexts I still find double-spaces at the end of a sentence preferable.  It's reeeeeeaaallyyy hard for me to remember not to do that for ES :-)

	"capture defense" 2.4
	"unplunderable" 1
	description "This rifle is commonplace among the Korath, predating the creation of the Repeater by at least two centuries. As it uses superheated plasma instead of lasers, the rifle is slow to fire and can sometimes burn its user's hands if they aren't careful. It makes up for these setbacks by guaranteeing that those hit won't live long enough to fire back."

outfit "Microbot Defense Station"
	category "Hand to Hand"
	cost 53000
	thumbnail "outfit/microbot defense station"
	mass 2
	"outfit space" -3
	"capture defense" 45
	atrocity 1
	"energy consumption" 0.01
	"heat generation" 0.1
	unplunderable 1
	description `This insidious trap awaits anyone foolish enough to attempt to capture Korath World-Ships. Intruders are trapped before this airlock, ahead of which a cloud of microbots programmed to break down organic matter into nutrigel is released. The resulting gel then drains into the vat below while the robots are recalled.`

> 	description `Many races find this means of defense abhorrent. Citing the excruciating, confusing death experienced by its victims, the violation of taboos relating to cannibalism, and erroneous fears that the micrbots could somehow "escape," mere possession of these airlocks may carry the death penalty.`

Reviewer 4: ```suggestion
	description `Many races find this means of defense abhorrent. Citing the excruciating, confusing death experienced by its victims, the violation of taboos relating to cannibalism, and erroneous fears that the microbots could somehow "escape," mere possession of these airlocks may carry the death penalty.`
```
